python/lsst/ts/xml/data/sal_interfaces/MTReflector/MTReflector_Events.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
+<SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
+  <Enumeration>
+    MTReflectorStatus_Connected=0,
+    MTReflectorStatus_Disconnected=1,
+    MTReflectorStatus_Open=2,
+    MTReflectorStatus_Close=3,
+    MTReflectorStatus_Unknown=4,
+    MTReflectorStatus_Connection_Error=5
+  </Enumeration>
+  <SALEvent>
+    <Subsystem>MTReflector</Subsystem>
+    <EFDB_Topic>MTReflector_logevent_reflectorStatus</EFDB_Topic>
+    <Description>MTReflector state change</Description>
+    <item>
+      <EFDB_Name>reflectorStatus</EFDB_Name>
+      <Description>MTReflector state; a ReflectorStatus enum.</Description>
+      <IDL_Type>int</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>MTReflector</Subsystem>
+    <EFDB_Topic>MTReflector_logevent_reflectorOpen</EFDB_Topic>
```

Comment:
I really don't think having different events for open and close is a good idea. You already have an event that indicates the status, so that is probably good enough. Please, remove these 2 events.